RTB_OP_CODES.s

```diff
@@ -113,7 +113,7 @@ TPMODE		CAF	ONE		# MODE IS TP.
 		TS	MPAC		# AND SKIP ON OVERFLOW.
 		TC	Q
 
-		INDEX	A		# OVERFLOW UNCORRECT AND IN MSU.
+		INDEX	A		# OVERFLOW INCORRECT AND IN MSU.
```

Comment:
This is `UNCORRECT` on the scan of [page 1398](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/1398.jpg).